protocol/test/forge/v2/strategies/RamosStrategy.t.sol

```diff
@@ -0,0 +1,458 @@
+    address public constant POOL_HELPER_ADDRESS = 0x4A02CbdBcd97BC639a13b864710550A7c39A3416;
+    AMO__IAuraStaking public amoStaking = AMO__IAuraStaking(0x70989E7D20C065ce7628C8DdAA240853437953d7);
+    IERC20 public bptToken = IERC20(0x8Bd4A1E74A27182D23B98c10Fd21D4FbB0eD4BA0);
+    IERC20 public dai = IERC20(0x6B175474E89094C44Da98b954EedeAC495271d0F);
+    IERC20 public aura = IERC20(0xC0c293ce456fF0ED870ADd98a0828Dd4d2903DBF);
+    IERC20 public bal = IERC20(0xba100000625a3754423978a60c9317c58a424e3D);
+
+    uint256 public constant defaultBaseInterest = 0.01e18;
+    TempleDebtToken public dUSD;
+    TreasuryReservesVault public trv;
+
+    address[] public reportedAssets = [address(dai), address(aura), address(bal)];
+
+    function _setUp() public {
+        fork("mainnet", 17300437);
+
+        RAMOS ramos = new RAMOS(balancerVault, temple, address(dai), address(bptToken), address(amoStaking), templeIndexPool, balancerPoolId);
+        vm.etch(RAMOS_ADDRESS, address(ramos).code);
+
+        PoolHelper poolHelper = new PoolHelper(balancerVault, balancerHelpers, temple, address(dai), address(bptToken), address(amoStaking), templeIndexPool, balancerPoolId);
+        vm.etch(POOL_HELPER_ADDRESS, address(poolHelper).code);
+
+        dUSD = new TempleDebtToken("Temple Debt", "dUSD", rescuer, executor, defaultBaseInterest);
+        trv = new TreasuryReservesVault(rescuer, executor, address(dai), address(dUSD), 9700);
+        strategy = new RamosStrategy(rescuer, executor, "RamosStrategy", address(trv), RAMOS_ADDRESS);
+
+        vm.startPrank(executor);
+        dUSD.addMinter(executor);
+        vm.stopPrank();
+    }
+}
+
+contract RamosStrategyTestAdmin is RamosStrategyTestBase {
+
+    function setUp() public {
+        _setUp();
+    }
+
+    function test_initalization() public {
+        assertEq(strategy.executors(executor), true);
+        assertEq(strategy.rescuers(rescuer), true);
+        assertEq(strategy.apiVersion(), "1.0.0");
+        assertEq(strategy.strategyName(), "RamosStrategy");
+        assertEq(strategy.strategyVersion(), "1.0.0");
+        assertEq(address(strategy.treasuryReservesVault()), address(trv));
+        assertEq(address(strategy.stableToken()), address(dai));
+        assertEq(address(strategy.internalDebtToken()), address(dUSD));
+        assertEq(strategy.manualAssetBalanceDeltas(address(dai)), 0);
+        assertEq(strategy.currentDebt(), 0);
+
+        assertEq(address(strategy.ramos()), RAMOS_ADDRESS);
+        address[] memory assets = new address[](0);
+        assertEq(strategy.getAssets(), assets);
+    }
+
+    function test_automatedShutdown() public {
+        vm.expectRevert(abi.encodeWithSelector(ITempleStrategy.Unimplemented.selector));
+        strategy.automatedShutdown();
+    }
+}
+
+contract RamosStrategyTestAccess is RamosStrategyTestBase {
+    function setUp() public {
+        _setUp();
+    }
+
```

Comment:
Missing testing the access for other `onlyElevatedAccess` functions in here -- eg borrowAndAddLiquidity, removeLiquidityAndRepay